subprojects/smoke-test/src/smokeTest/groovy/org/gradle/smoketests/AbstractGradleceptionSmokeTest.groovy

```diff
@@ -46,6 +46,7 @@ abstract class AbstractGradleceptionSmokeTest extends AbstractSmokeTest {
         and:
         def buildJavaHome = AvailableJavaHomes.getAvailableJdks(new GradleBuildJvmSpec()).last().javaHome
         file("gradle.properties") << "\norg.gradle.java.home=${buildJavaHome}\n"
+        file("gradle.properties") << "systemProp.org.gradle.unsafe.kotlin.assignment=true\n"
```

Comment:
❗ This looks like something that can be easily left here. Would be nice to add some comment at least for someone bumping on it later to know if they can remove it.

Reply:
Good idea.

Reply:
Added a comment